sql/views.sql

```diff
@@ -69,6 +69,23 @@ CREATE OR REPLACE VIEW timescaledb_information.policy_stats as
     INNER JOIN _timescaledb_internal.bgw_job_stat js on j.id = js.job_id
   ORDER BY ht.schema_name, ht.table_name;
 
+-- view for background worker jobs
+CREATE OR REPLACE VIEW timescaledb_information.jobs AS
+SELECT j.id AS job_id,
+  j.application_name,
+  j.schedule_interval,
+  j.max_runtime,
+  j.max_retries,
+  j.retry_period,
+  j.proc_schema,
+  j.proc_name,
+  j.owner,
+  j.scheduled,
+  j.config,
```

Comment:
I notice we're writing the entire jobs table except `hypertable_id`.  While I agree that field isn't going to be publicly useful, it might be useful to join in the hypertable table and include the table name and schema.

Reply:
hypertable_id is completely internal and may point to internal hypertables

Reply:
Hmm, I do still think including the table name would be useful, though definitely less so when it points to an internal table.  Right now if a user creates two policies with the same parameters, there is no way for them to tell from this new view which job is for which table.

Reply:
in case of caggs this points to the materialization hypertable, i guess showing the name would be nice but what would you call the columns? for caggs we should resolve to the cagg view name which means we couldnt call the column hypertable_schema/name

Reply:
I think it's actually okay to use the materialization hypertable for caggs as the continuous_aggregates information view also shows the materialization_hypertable, and can be used to match the job to the cagg view name.

Reply:
ok added hypertable name and schema